code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -0,0 +1,37 @@
+/* global React, dashboard */
+
+var STAGE_PROGRESS_TYPE = require('./stage_progress_type');
+var StageProgress = require('./stage_progress');
+
+/**
+ * Stage progress component used in level header and course overview.
+ */
+var CourseProgressRow = React.createClass({
+  propTypes: {
+    stage: React.PropTypes.shape({
```

Comment:
super-minor nit: Is there somewhere we could record this shape for reference? I notice you're using it both here and in `course_progress.jsx`, and it would be nice to not be.